### PR TITLE
WIP: New PharmGKB importer

### PR DESCRIPTION
**Pull request by kcotto**

### CODE REVIEW

File: lib/genome/importers/pharmgkb/get_genes_pharmgkb.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+with open(snp_file, 'r') as snp:
+    reader = csv.DictReader(snp, delimiter='\t')
+    header = reader.fieldnames
+    for line in reader:
+        snp_dict[line['Variant Name']] = line['Gene Symbols']
+
+with open(pharmgkb_file, 'r') as pharmgkb, open('PharmGKB_clean.tsv', 'w') as tmp:
+    pharmgkb_reader = csv.reader(pharmgkb, delimiter='\t')
+    tmp.write('\t'.join(['GeneID', 'Gene', 'ChemicalID', 'Chemical', 'Evidence', 'Association', 'PK', 'PD', 'PMIDs', 'VariantID']) + '\n')
```
Comment (susannasiebert) [inline lib/genome/importers/pharmgkb/get_genes_pharmgkb.py]: I would use a `csv.writer` here

**Comment by susannasiebert** [comment]
Are we planning on making this an automated updater? If so, we'll need to figure out a way to download the TSVs automatically.

**Comment by kcotto** [comment]
Yeah. This will need reworking to be an online updater. I was just working off of the fact that Malachi gave me the interaction tsv. We'll need his credentials to work on the automatic updating.

**Comment by ahwagner** [comment]
@kcotto to revive this, auto-download tsv/csv

**Comment by ahwagner** [comment]
changing to WIP, pending resolution of https://github.com/griffithlab/dgi-db/pull/314#issuecomment-625357730

**Comment by susannasiebert** [comment]
Closing in favor of #425

**Pull request closed without merging**